src/parseData.js

```diff
@@ -0,0 +1,55 @@
+const fs      = require('fs');
+const request = require('request');
+const cheerio = require('cheerio');
+
+request('http://www.covidmaroc.ma/Pages/AccueilAR.aspx', function(error, response, html) {
+	if(!error) {
+		var $ = cheerio.load(html);
+
+		var confirmed, deaths, recovered, notConfirmed, date;
+		var json = require('./databackup.json');
```

Comment:
Why do you need this file here?

Reply:
because of the position of region markers. think of that file as a template I only change the values and then dump it to data.json

Reply:
In my opinion, we should have a file called regions.json that has all the regions and locations as key > value and use it to render the markers on the map.
So that data.json will only contain data fetched by your script.

Reply:
but then I will have to make a dictionary for each region when I am scraping the data. the reason why I did it like that is because I don't know whether they will change the order of the regions so what I am doing instead is comparing the string scraped with the one that is already there.

Reply:
It's not that it's impossible but if you want then sure.

Reply:
I think I'm missing something, even if the order of the table changes that shouldn't be a problem.

I imagine something like this:
- Parse the HTML table so that we have an array of {region, count}
- Dump it as it is to the data.json file (in locations)
- When rendering the map we loop through each region get its coordinates from regions.json and render it

Can you explain to me how its working in your code please?

Reply:
In the original code. We're already looping on Locations and putting down markers with radius being the count variable. so to avoid extra code, I just loop through it again and update the count variable. simple as that.

Reply:
This feels like a recoding session :)

Reply:
Oh Ok I just counted the regions in the official website, its 12 so all regions are there.
I was thinking that when they will add regions it will mess up everything.
Good job!

It's not a recoding session, just want to make sure everything will work fine.

Reply:
It's okay. I hope that I made everything clear.